wire-ios-sync-engine/Source/Calling/MLSConferenceStaleParticipantsRemover.swift

```diff
@@ -48,9 +48,29 @@ class MLSConferenceStaleParticipantsRemover: Subscriber {
         self.removalTimeout = removalTimeout
     }
 
+    deinit {
+        subscription?.cancel()
+        subscription = nil
+    }
+
+    private var subscription: Subscription? {
+        didSet {
+            stopSubscribing = false
+        }
+    }
     // MARK: - Subscriber implementation
 
+    var stopSubscribing: Bool = false {
+        didSet {
+            if stopSubscribing {
+                subscription?.cancel()
+                subscription = nil
+            }
+        }
+    }
+
     func receive(subscription: Subscription) {
+        self.subscription = subscription
         subscription.request(.unlimited)
     }
```

Comment:
Is there a point to using `didset` here?
Maybe I'm missing something, but wouldn't this be simpler?


```suggestion
    deinit {
        stopSubscribing()
    }

    private var subscription: Subscription? 
    
    // MARK: - Subscriber implementation
    
    func stopSubscribing() {
        subscription?.cancel()
        subscription = nil
    }

    func receive(subscription: Subscription) {
        self.subscription = subscription
        subscription.request(.unlimited)
    }
```